src/main/java/com/fauna/codec/codecs/BaseRefCodec.java

```diff
@@ -7,8 +7,6 @@
 import com.fauna.exception.CodecException;
 import com.fauna.types.*;
```

Comment:
You fixed some of these in this PR. Maybe you want to do this one too?

Reply:
Will fix in a future PR.